docs/advanced-transfer/experimental-features.mdx

```diff
@@ -29,8 +29,18 @@ The `experimental_features` parameter accepts an array of strings, where each st
 
 **Identifier:** "eureka"
 
+### Layerzero (`"layer_zero"`)
+
+**Description:**  
+Improves cross-chain capabilities to support more assets such as USDT.
```

Comment:
I think this should look more like the stargate one (where it specifies the bridge, includes some chains and assets)